src/schema/objects/entities.yaml

```diff
@@ -53,6 +46,14 @@ ceagent:
     `"ContrastBolusIngredient"` MAY also be added in the JSON file, with the same label.
   type: string
   format: label
+cohort:
+  name: cohort
+  display_name: Cohort
+  description: |
+    A subset of a defined template space, for instance, for a longitudinal template of brain development
+    where infants were participants were averaged at three, six, and twelve months old.
```

Comment:
```suggestion
    A sub-population over which an atlas or template is derived.
```

Reply:
I've removed the example, because that didn't seem like a cohort to me.  In my mental model is a longitudinal study is one or more cohorts with observations at multiple timepoints.

Reply:
Well, then, maybe we need to find an alternate entity name because the term 'cohort' comes from TemplateFlow, and there, we do use it to separate different ages within a single template/atlas. It's also the intention here: to have a single standard space name that, in reality, has several standard spaces within.

Reply:
Oh sorry, I misunderstood. I agree with the merriam webster definition and now understand the longitudinal example.  I had originally though the example was referring to multiple *timepoints* at 3, 6 and 12 months old.

Reply:
Maybe the definition could be more clear, please send suggestions if you think we can make it better :)

Reply:
My suggested change is above.  I suggest changing the definition to "A sub-population over which an atlas or template is derived." and removing the example.